### DISCUSSION TITLE
Accounting for anisotropy in seismic inversion.

### DISCUSSION THREAD

**Discussion by RobertZetica** [gimli-org/pyGIMLi#735] [General]
Hi there, is it possible to account for the effect of anisotropy when using the TravelTimeManager "Invert" function?

**Comment by halbmy** [comment]
No, unfortunately not. The algorithm currently supports only isotropic slowness, but we are aware that this feature is of interest for a wide range of applications. 

Anybody with experiences in anisotropic traveltime is highly welcomed to be part of the discussion.
An inversion would then invert for a main slowness and an anisotropy factor using either prior knowledge on the angle or still another parameter distribution.

For a globally known anisotropy, one could just deform the whole geometry (sensor positions and mesh) in such a way that it is stretched with the inverse anisotropy factor. Did anyone try that?

**Comment by makeabhishek** [comment]
I'm also interested to perform anisotropy tomography and I'll be happy to be part to develop it under the guidance.

**Comment by halbmy** [comment]
Does anyone have a suitable dataset or a benchmark model? 

First step could be to deform the mesh globally using a known anisotropy factor. Second could be to invert also for this factor, then to do this based on subsurface regions.

A general solution (spatially distributed anisotropy) would require changing the C++ core for solving the Eikonal equation (by multiplying  the traveltime with a correction factor) but more importantly for deriving the Jacobian with respect to the anisotropy factor. Is there any paper already describing the theory? I know colleagues working on the subject.

**Reply by makeabhishek** [reply]
I don't have a benchmark data. But I did finite element simulations earlier to generate time traces. It's is transversely isotropic. But I can also generate data for fully anisotropic.

**Reply by makeabhishek** [reply]
What kind of dataset is needed in that case?

**Reply by halbmy** [reply]
Of course a benchmark dataset helps, but I am missing a paper with the theory for the implementation.

**Reply by makeabhishek** [reply]
Here are the papers I found. 

1. [First-arrival traveltime tomography for anisotropic media using the adjoint-state method](https://library.seg.org/doi/full/10.1190/geo2015-0463.1)
2. "[Travel times and ray paths for acoustic and elastic waves in generally anisotropic media.](https://www.sciencedirect.com/science/article/pii/S0021999123005958?ref=pdf_download&fr=RR-2&rr=8dd1c7ee79f6e74d)"
